main/kamailio/APKBUILD

```diff
@@ -275,7 +272,7 @@ prepare() {
 
 build() {
 	cd "$_builddir"
-	make FLAVOUR=kamailio STUN=1 \
+	make -j16 FLAVOUR=kamailio STUN=1 \
```

Comment:
Is this needed? Normally abuild.conf sets JOBS and MAKEFLAGS to indicate proper parallelism level.